### ISSUE TITLE
x-axsis Unit problem

### ISSUE THREAD

**Issue by blue14753** [wcharczuk/go-chart#142]
Hi,

Is there any way to show the x-axsis by a fixed number I want ,
like 30 60 90 120 150 (I want 30 ,and maybe x input is 38 ,will between 30-60 )

not the x input's average,like min=0,max=90, and 5 item => 0 18 36 54 72 90

thanks.

**Comment by PradyumnaShome** [comment]
If you set the Ticks variable for the XAxis, it will use your ticks instead of auto-generated ones.

**Comment by drgrib** [comment]
Yeah, but what if you want to remove them completely as he asked? If I set `Ticks: nil`, it just uses the default ticks.
